action/protocol/poll/consortium.go

```diff
@@ -50,10 +50,11 @@ type consortiumCommittee struct {
 	indexer        *CandidateIndexer
 	addr           address.Address
 	getBlockHash   evm.GetBlockHash
+	getBlockTime   evm.GetBlockTime
 }
 
 // NewConsortiumCommittee creates a committee for consorium chain
-func NewConsortiumCommittee(indexer *CandidateIndexer, readContract ReadContract, getBlockHash evm.GetBlockHash) (Protocol, error) {
+func NewConsortiumCommittee(indexer *CandidateIndexer, readContract ReadContract, getBlockHash evm.GetBlockHash, getBlockTime evm.GetBlockTime) (Protocol, error) {
```

Comment:
it seems it's only used for genesis block, so we can know Shanghai is not enabled, no need to pass in a real func